src/lib/krb5/ccache/cc_mslsa.c

```diff
@@ -787,18 +794,14 @@ KerbSubmitTicket( HANDLE LogonHandle, ULONG  PackageId,
         krb5_auth_con_setsendsubkey(context, auth_context, keyblock);
     }
     rc = krb5_mk_1cred(context, auth_context, cred, &krb_cred, &replaydata);
-    if (rc) {
-        krb5_auth_con_free(context, auth_context);
-        if (keyblock)
-            krb5_free_keyblock(context, keyblock);
-        if (krb_cred)
-            krb5_free_data(context, krb_cred);
-        return FALSE;
-    }
+    if (rc)
+        goto cleanup;
```

Comment:
`pSubmitRequest` needs to be initialized `NULL` otherwise this (and the next `goto cleanup`) cause an unitialized `free()`.